File: app/src/main/java/br/com/disapps/homepet/util/rx/FormError.java
package br.com.disapps.homepet.util.rx;

import java.util.ArrayList;
import java.util.HashMap;

/**
 * Created by diefferson.santos on 23/08/17.
 */

public class FormError extends HashMap<String, ArrayList<String>> {

    public String getFirstError(String key){
        return this.get(key).get(0);
    }

}

File: app/src/main/java/br/com/disapps/homepet/util/rx/FormError.kt
package br.com.disapps.homepet.util.rx

import java.util.ArrayList
import java.util.HashMap

/**
 * Created by diefferson.santos on 23/08/17.
 */

class FormError : HashMap<String, ArrayList<String>>() {

    fun getFirstError(key: String) = this[key]!!.get(0)

}
